api-docs/swagger.yml

```diff
@@ -1,31 +1,29 @@
 swagger: '2.0'
 info:
   description: OpenFaaS API documentation
-  version: 0.8.1
+  version: 0.8.2
   title: OpenFaaS API Gateway
   license:
     name: MIT
 basePath: /
 schemes:
   - http
+  - https
```

Comment:
I'm not sure that the gateway has a scheme for `https`, I'd remove that. You can add a TLS proxy in front of any service, but it - itself cannot serve TLS.

Reply:
Removed it in 5c657e6